qutebrowser/misc/crashdialog.py

```diff
@@ -30,13 +30,13 @@
 import enum
 import typing
 
-import pkg_resources
 from PyQt5.QtCore import pyqtSlot, Qt, QSize
 from PyQt5.QtWidgets import (QDialog, QLabel, QTextEdit, QPushButton,
                              QVBoxLayout, QHBoxLayout, QCheckBox,
                              QDialogButtonBox, QApplication, QMessageBox)
 
 import qutebrowser
+from qutebrowser.utils.qtutils import parse_version
```

Comment:
Please import modules instead of things from modules (so that it's clear when reading the code where stuff is coming from). Also see my comment below about moving `parse_version`, though.